src/main/java/nl/novi/techItEasy/models/Television.java

```diff
@@ -30,13 +30,18 @@ public class Television {
     private Integer sold;
 
     // Cascade komt uit EdHub H14, volgens mij zorgt dat ervoor dat je bij een delete ook de gerelateerde info delete
-    // In huiswerkles werd private voor de variabele gezet. In H14 niks (public?)
+    // In huiswerkles werd private voor de variabele gezet. In H14 niks (package private)
```

Comment:
Ik raad wel aan om in Entity-klasses altijd met private variabelen te werken en getters en setters. Zo voorkom je dat variabelen direct aangesproken (en aangepast) kunnen worden